chronicles/options.nim

```diff
@@ -289,5 +289,8 @@ const
 
   config* = when chronicles_streams.len > 0: parseStreamsSpec(chronicles_streams)
             elif chronicles_sinks.len > 0:   parseSinksSpec(chronicles_sinks)
-            else: parseSinksSpec "textblocks"
+            # default is textlines - single-line format is good for interop with
+            # text processing tools like grep and logstash, good use of screen
+            # real estate
+            else: parseSinksSpec "textlines"
```

Comment:
Since the team decided on this change, that's OK, but I'll voice few more arguments for the former textblocks choice:

* The argument that log systems are already optimized to deal with one record per-line is not that strong IMO, because most systems already had to deal with stack traces appearing in the logs and I think a lot of specialized software has adopted the rule that indented blocks are considered part of the log line before them.

* The gripability argument is not that strong either, because most people don't know about the `-A` and `-B` options of grep, which solves the problem to a reasonable extent. I have quite a lot of experience in grepping the indented human-readable output of https://github.com/trentm/node-bunyan (one of the most popular and established json loggers).

* The dense log produced by `textlines` doesn't come without downsides too:
  - Long lines will be wrapped making them harder to read. This creates a psychological effect where the developer is incentivized to log less information in order to keep the log readable.
  - Visually scanning a log line to find a particular property value will be objectively slower.

Reply:
documented these arguments as well.

>  I think a lot of specialized software has adopted the rule that indented blocks are considered part of the log line before them
this is an interesting point - examples of such systems? potentially, these might come from the python world where indents are the norm. I checked logstash - it has a special plugin that can deal with this by specifying a regex to match: https://www.elastic.co/guide/en/logstash/current/multiline.html - however, this is not part of the "default" config and requires extra setup

> gripability argument
two things of note:
* `-A xx -B xx` - there's no way to choose good numbers here because log lines have different numbers of params and you don't know which one you're grepping for - also, like you say, these arguments are less well known, so the `out-of-the-box` experience is worse
* it's also easier to regex-match a single line - you can do `INF.*message here.*ip=XXX` to get a quick `and` filter.. the same holds true for IDE/editor searches etc